tests/cluster/delta/deltaspec/test_tblspec.py

```diff
@@ -0,0 +1,67 @@
+import unittest
+
+from spetlrtools.testing import DataframeTestCase
+
+from spetlr import Configurator
+from spetlr.spark import Spark
+from tests.cluster.delta.deltaspec import tables
+
+
+class TestTableSpec(DataframeTestCase):
```

Comment:
do you test some of the places where you expect raises?

Reply:
I added tests